### ISSUE TITLE
[feature_request] Decrease joint trajectory controller goal tollerance

### ISSUE THREAD

**Issue by rickstaa** [frankaemika/franka_ros#201]
As mentioned in https://github.com/frankaemika/franka_ros/commit/e899240365797de6d53c222d6512b74c3a40268e#commitcomment-60095175 the current goal tolerance of the JointTrajctory controller is quite big. I think making it smaller (~`0.005`) would enforce more accurate behaviour:

https://github.com/frankaemika/franka_ros/blob/e899240365797de6d53c222d6512b74c3a40268e/franka_control/config/default_controllers.yaml#L13-L19

https://github.com/frankaemika/franka_ros/blob/e899240365797de6d53c222d6512b74c3a40268e/franka_control/config/default_controllers.yaml#L41-L47

**Comment by fe-support** [comment]
Closing due to inactivity. Please open this git issue again if the topic still exists.

**Comment by rickstaa** [comment]
@fe-support I think the concern given by @rhaschke still is valid as @frankaemika did not change the contain tolerance:

https://github.com/frankaemika/franka_ros/blob/358406952597c3735d43625595650901be84a5d8/franka_control/config/default_controllers.yaml#L11-L19

I think closing without explaining why the big tolerance is given might not be the best option 🤔. Can you maybe re-open this issue or explain why you chose the `5cm` tolerance?

**Comment by marcbone** [comment]
yeah we shouldnt have closed this. Nobody really investigated this so I am gonna reopen it. Also its not 5 cm but 0.05 radian :sweat_smile:
